.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor
 composer.lock
+code-coverage
```

Comment:
Could you add a newline here? Reasons are listed here: http://stackoverflow.com/q/729692/871913